static/js/chart/draw.ts

```diff
@@ -431,42 +436,77 @@ function drawLineChart(
 
 
 /**
- * Draw a group of line charts with different goeId.
+ * Return all styles information for given dataGroupsDict.
+ * {
+ *     colors: [],
+ *     dashes: [],
+ *     statVars: [],
+ *     geoIds: [],
+ *     maxV: ,
+ * }
  *
- * @param id
- * @param width
- * @param height
  * @param dataGroupsDict
- * @param unit
  */
-function drawGroupLineChart(
-    id: string,
-    width: number,
-    height: number,
-    dataGroupsDict: {[geoId: string]: DataGroup[];},
-    unit?: string
+function generateStyles(
+    dataGroupsDict: {[geoId: string]: DataGroup[] },
 ) {
-  console.log(dataGroupsDict);
+  let styles = {};
+  styles["colors"] = [];
+  styles["dashes"] = [];
+  styles["statVars"] = [];
```

Comment:
Consider renaming this to statVarLabels for clarity

Reply:
Yep, already renamed it~